pyof/utils.py

```diff
@@ -61,3 +62,24 @@ def unpack(packet):
         return message
     except (UnpackException, ValueError) as exception:
         raise UnpackException(exception)
+
+
+def is_ofbac_bad_out_port(code):
+    """Check if the code is a OFPBAC_BAD_OUT_PORT error."""
+    errors = (v0x01.asynchronous.error_msg.BadActionCode.OFPBAC_BAD_OUT_PORT,
+              v0x04.asynchronous.error_msg.BadActionCode.OFPBAC_BAD_OUT_PORT)
+    return code in errors
+
+
+def get_port_config_for_version(version):
+    """Return port_config object to a specific version of python-openflow."""
+    port_config = None
+
+    if version == 0x01:
+        port_config = v0x01.common.phy_port.PortConfig.OFPPC_NO_FWD
+    elif version == 0x04:
+        port_config = v0x04.common.port.PortConfig.OFPPC_NO_FWD
+    else:
+        raise UnsupportedVersionException(version)
```

Comment:
I'm now thinking that maybe none of this code is necessary. I've just checked the spec for 1.4 and 1.5 and they are all the same values from 1.0/1.3, and highly unlikely to change in the future, so we can eliminate this complexity and just use the values same Enum values from 1.0. What do you think, @josemauro?

Reply:
That's a good point. Thanks! I'll close this PR and add your observation in the pr https://github.com/kytos/flow_manager/pull/92